pyvista/plotting/widgets.py

```diff
@@ -651,6 +715,132 @@ def callback(normal, loc):
             return self.add_mesh(crinkler, **kwargs)
         return self.add_mesh(clipper, **kwargs)
 
+    def add_volume_clip_plane(
+        self,
+        volume,
+        normal='x',
+        invert=False,
+        widget_color=None,
+        value=0.0,
+        assign_to_axis=None,
+        tubing=False,
+        origin_translation=True,
+        outline_translation=False,
+        implicit=True,
+        normal_rotation=True,
+        interaction_event='end',
+        origin=None,
+        **kwargs,
+    ):
+        """Clip a volume using a plane widget.
+
+        Parameters
+        ----------
+        volume : pyvista.plotting.Volume, pyvista.UniformGrid, pyvista.RectilinearGrid
```

Comment:
are comma delimitors supported? we usually use `or` to separate supported types

Reply:
According to https://numpydoc.readthedocs.io/en/latest/format.html, it looks like 'or' is prefered.